Tasks/AzureAppServiceManage/task.json

```diff
@@ -239,8 +239,8 @@
         "FailedtoRestartAppService": "Failed to restart App Service for '%s'. Response Code : %s (%s)",
         "RestartAppServiceAccepted": "Restart request accepted by App Service : %s",
         "InvalidAction": "Invalid Action selected !",
-        "WARNINGCannotupdatedeploymentstatusSCMendpointisnotenabledforthiswebsite": "WARNING: Cannot update deployment statu. SCM endpoint is not enabled for this website.",
-        "CouldnotfetchacccesstokenforAzureStatusCode": "Could not fetch acccess token for Azure App Service. StatusCode %s (%s)",
+        "WARNINGCannotupdatedeploymentstatusSCMendpointisnotenabledforthiswebsite": "WARNING: Cannot update deployment status. SCM endpoint is not enabled for this website.",
+        "CouldnotfetchaccesstokenforAzureStatusCode": "Could not fetch acccess token for Azure App Service. StatusCode %s (%s)",
```

Comment:
What is the change for this resource key? I see bunch of resjson having just this change...